airship/kubeadm-airship-gap-analysis.md

```diff
@@ -112,13 +112,58 @@ Jira Issues:
 # RunTimeClass configuration
 Jira issues:
 - [RunTimeClass configuration](https://airship.atlassian.net/browse/AIR-141)
+## RuntimeClass Introduction
+It is possible to define RunTimeClass kind configuration to control runtimes used for pods.
+See example runtimeclass definition below.
+```
+apiVersion: node.k8s.io/v1beta1
+kind: RuntimeClass
+metadata:
+  name: containerdruntimeclass
+handler: containerd
+```
+After class is configured, it can be taken into use in pods, see example pod definition below.
+```
+apiVersion: v1                                                               
+kind: Pod                                                                    
+metadata:                                                                    
+  name: example                                                              
+spec:                                                                        
+  runtimeClassName: containerdruntimeclass
+  containers:                                                                
+    - name: dummy-pod                                                        
+      image: ubuntu                                                          
+      command: ["/bin/bash", "-ec", "while :; do echo '.'; sleep 5 ; done"]  
+  restartPolicy: Never                                                       
+```
+## Runtime selection in Kubeadm
+Kubeadm detects available runtime sockets and if there is more than one runtime socket available, kubeadm expects
+that user will define wanted runtime by setting --cri-socket parameter.
+See example below
+```
+kubeadm init --cri-socket /var/run/crio/crio.sock
+```
+
+## Runtime installation
+Packages needed for runtime installations can be installed into ISO/qcow2 images.
+Configuration of runtime for Kubelet can be done in cloud init phase before Kubeadmin execution.
+See example below how to configure kubelet to use selected runtime:
+
+```
+cat > /etc/systemd/system/kubelet.service.d/20-cri.conf <<EOF
+[Service]
+Environment="KUBELET_EXTRA_ARGS=--container-runtime=remote --container-runtime-endpoint=containerd"
+EOF
+systemctl daemon-reload
+``
```

Comment:
There is a problem on this line

the next sections are malformed because of the missing bak tick (```) you need three

Reply:
Done